src/screens/Login/index.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import './style.css';
+import '../../style.css';
+import wbooks from '../../assets/wbooks_logo.svg';
+import FormErrors from './components/FormErrors';
+import { Redirect } from 'react-router-dom'; 
+import axios from '../../config/api'
+
+class Login extends Component {
+  state = {
+    email: '',
+    pass: '',
+    formErrors: {email: '', pass: ''},
+    emailValid: false,
+    passValid: false,
+    formValid: false,
+    login: false
+  }  
+  
+  handleUserInput = (e) => {
+    const name = e.target.name;
+    const value = e.target.value;
+    this.setState({[name]: value},
+                   () => { this.validateField(name, value) 
+                  });              
+  }
+
+  validateField(fieldName, value) {
+    let fieldValidationErrors = this.state.formErrors;
+    let emailValid = this.state.emailValid;
+    let passwordValid = this.state.passwordValid;
+  
+    switch(fieldName) {
+      case 'email':
+        emailValid = value.match(/^([\w.%+-]+)@([\w-]+\.)+([\w]{2,})$/i);
+        fieldValidationErrors.email = emailValid ? '' : 'El email ingresado no tiene un formato correcto';
+        break;
+      case 'pass':
+        passwordValid = (value.length >= 8) & (value.length <= 52);
+        fieldValidationErrors.password = passwordValid ? '': 'La contraseña debe tener entre 8 y 54 caracteres';
+        break;
+      default:
+        break;
+    }
+    this.setState({formErrors: fieldValidationErrors,
+                    emailValid: emailValid,
+                    passwordValid: passwordValid
+                  }, this.validateForm);
+  }
+
+  validateForm() {
+    this.setState({formValid: this.state.emailValid && this.state.passwordValid});
+  }
+
+  handleSubmit = (event) => {
+      event.preventDefault();
+      axios.post('/users/sessions', {
```

Comment:
We use service for requests.
In this case the service should make the request and save the token.
So I expect `handleSubmit` to look something like this:


```js
AuthService.login(this.state.email, this.state.pass).then(() => {
  this.setState({login: true});
}).catch(() => {
  const formErrors = {user: 'Pass incorrect'};
  this.setState({formErrors }, this.validateForm); 
});